extensions/processing/README.md

```diff
@@ -0,0 +1,64 @@
+# Processing Extension Specification
+
+- **Title: Processing**
+- **Identifier: proc**
+- **Field Name Prefix: proc**
+- **Scope: Item, Collection**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+Processing metadata is considered to be data that indicate from which processing chain a data originates and how the data itself has been produced. Overall, it helps to increase traceability and search among processing levels and multiple algorithm versions.
+
+Often, data items are the result of one or more waterfall processing pipeline. Tracing information such as the processing facility , the algorithm version or the processing date helps in the data version management.
+
+This extension applies to STAC Items and STAC Collections. As these processing information are often closely bound to the collection level and therefore are shared across all items, it is recommended adding the fields to the corresponding STAC Collection.
+
+- [Examples](examples/): [Sentinel-1 Item](examples/sentinel1-item.json),  [Sentinel-2 Level 1C Collection](examples/sentinel2-collection.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Item and Collection fields
```

Comment:
We have recently also discussed with @prateekbudhwar and colleagues to add processing details to STAC (outdated PR #881). The idea we had was to simply add a `lineage` free-text field to the providers array in Collections so that you can track the processing for each entity that has processed date. Would you think that makes also sense for your fields?

Example:
```
{
  "stac_version": "1.0.0-beta.2",
  "id": "...",
  "description": "...",
  "license": "proprietary",
  "providers": [
    {
      "name": "European Union/ESA/Copernicus",
      "roles": [
        "producer",
        "licensor"
      ],
      "proc:lineage": "Atmospheric Correction..."
    },
    {
      "name": "SomeComp",
      "roles": [
        "processor"
      ],
      "proc:lineage": "Foo bar... NDVI processing..."
    },
    {
      "name": "Amazon",
      "roles": [
        "host"
      ],
      "proc:lineage": "Conversion to COG..."
    }
  ],
  "extent": {...}
}
```

I think for Collections it may make sense to push your suggested fields into the providers array like proc:lineage in this example. Could be overkill for items and we just put it into properties there. Thoughts?

Anyway, could we add a field proc:lineage to the proposal? Free-text string.

Reply:
Indeed, that's the common idea with this processing extension. So I totally see lineage could be part of this proposal. My only doubt is about having it at assets level. For us, an item is an atomic product even if split of several assets and thus should share the same mileage. For instance, a sentinel-2 product can have 13 bands that have all been processed during the same "lineage" and thus 13 assets all having the "proc:lineage": "Atmospheric Correction..." property would be overkill IMO.

Reply:
Your proposal was going to be mainly for properties in items, right? I think that is correct and I didn't meant to say something different here. You could put them in assets, like any other property, but I doubt providers will do that if not absolutely required. So I'm fine with it.

Reply:
+1 for proc:lineage 

And defining it at the Item property level makes sense to me, and then people could choose to use it at the asset level if they really wanted.

Re-using proc:lineage at the provider level for Collections also makes sense to me.

Reply:
I have replaced `processing:name` with processing:lineage` referring to https://wiki.earthdata.nasa.gov/display/NASAISO/Lineage+Information.
For the moment, I have left all processing properties at the same level extending to the asset. I don't know exactly how to specify the extension with providers.
Feel free to suggest other places where they could be specified. On my side, I am very attach to the formalism 1 item = 1 dataset but there are probably several other use cases.

Reply:
I can do the Collection work for you and also do the JSON Schema. :-)